src/main/kotlin/no/skatteetaten/aurora/gobo/service/application/applicationResources.kt

```diff
@@ -22,29 +27,31 @@ data class PodResourceResource(
     val status: String,
     val restartCount: Int,
     val ready: Boolean,
-    val startTime: Instant
-) : ResourceSupport()
+    val startTime: Instant,
+    override val links: List<Link>
```

Comment:
Hvorfor bruker vi ikke ResourceSupport?
Og hvorfor override vi links?

Testene feiler fordi jackson må konfigureres opp til å lese "_links", dette gjør ResourceSupport for oss.

Reply:
ResourceSupport ser ikke ut til deserialisere linker fra Mokey, så jeg gikk bort fra det. Har dette fungert tidligere?

Reply:
Jeg trodde det fungert ja, men kanskje jeg ikke har testet det godt nok.
Skal gjøre noen tester i master branchen.

Reply:
Jeg kjørte testen ApplicationServiceTest `Get applications for affiliation`.
Når jeg debugger svaret, som kommer fra en stub generert i mokey med _links, så får jeg 2 linker under applicationInstances.

For meg ser det ut til å fungere, men du har kanskje gjort en annen test hvor det ikke fungerte?

Reply:
Det ser ikke ut til at mokey leverer linkene med "_links" (men som "links"). Da vil ikke ResourceSupport fange dem opp. Dette skjer selv om jeg setter Accept: application/hal+json og mokey responderer med Content-Type: application/hal+json;charset=UTF-8. Jeg catchet ikke dette i forrige runde. Tar en titt på mokey i morra.

Reply:
Dette er litt rart, for hvis jeg sjekker feks denne pathen i browseren: /api/application?affiliation=...
Så får jeg "_links".
Hvilken url er det du har sett på?